src/adv.py

```diff
@@ -38,6 +39,38 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player('1', room['outside'])
+
+move_options = ['n', 's', 'e', 'w', 'q'] 
+def get_move():
+    mv = input('Where would u like to go? -> Type n, s, e, or w: ')
+    if mv not in move_options:
+        print('Invalid direction')
+        return player.location
+    else:
+        if mv == 'n' and hasattr(player.location, 'n_to'):
+            next_room = player.location.n_to
+            #print(mv)
+            return(next_room)
+        elif mv == 's' and hasattr(player.location, 's_to'):
+            next_room = player.location.s_to
+            #print(mv)
+            return(next_room)
+        elif mv == 'e' and hasattr(player.location, 'e_to'):
+            next_room = player.location.e_to
+            #print(mv)
+            return(next_room)
+        elif mv == 'w' and hasattr(player.location, 'w_to'):
+            next_room = player.location.w_to
+            #print(mv)
+            return(next_room)
+        elif mv == 'q':
+            print('Deuces!')
+            exit()
+        else:
+            print("Can't go that way")
+            return player.location
+
```

Comment:
Well done using `hasattr` and I nice job abstracting `get_move()` to a function.